### DISCUSSION TITLE
Azure AKS Limitation - Burst VMs

### DISCUSSION THREAD

**Discussion by Chris530906** [QualiTorque/qualitorque.github.io#244] [Documentation requests] [answered]
Per DBS, Torque does not support running the Torque Agent on an AKS cluster using Burst Type VMs as cluster worker nodes. Due to stability reasons and limitations of the burst type VM on Azure we cannot reliably expect the agents to be available. Please update the AKS Torque Agent onboarding documentation to warn users of this and suggest a different VM type.

Documentation Locations: 
https://docs.qtorque.io/admin-guide/authentication/service-accounts-for-azure
https://docs.qtorque.io/getting-started/Install-and-connect-self-hosted-agent

Azure VM type: https://learn.microsoft.com/en-us/azure/virtual-machines/sizes-b-series-burstable

**Comment by Shnitselon** [comment, marked as answer]
Thanks a lot @Chris530906.

Updated the authentication article
